docs/pages/eas-build-custom-builds/custom-build-config.mdx

```diff
@@ -10,7 +9,11 @@ import ImageSpotlight from '~/components/plugins/ImageSpotlight';
 import { BoxLink } from '~/ui/components/BoxLink';
 import { GithubIcon } from '@expo/styleguide-icons';
 
-> **Warning** All content in this document is experimental, under active development, and likely to change. Use at your own risk. If you're looking to execute custom code before or after certain steps in the build process, but don't need to actually modify any of the steps, you may want to use the stable [build lifecycle hooks API](/build-reference/npm-hooks/).
+> **Warning** This functionality is in preview, under active development, and it can be subject to changes.
+
+{/* TODO: link to the blog post */}
+
+> **Warning** Other, current limitations of custom builds are highlighted in the launch blog post and include: not adapted pricing, limitations regarding uploading artifacts, lack of reusable steps for caching dependencies, no support for classic updates, and not ready GitHub integration for custom builds.
```

Comment:
```suggestion
> **Warning** Current limitations of custom builds are highlighted in the [changelog](https://expo.dev/changelog/2023/08-10-custom-build-jobs) and include not adapted pricing, limitations regarding uploading artifacts, lack of reusable steps for caching dependencies, no support for classic updates, and not ready GitHub integration for custom builds.
```

Added the changelog post here since the blog post will also be copy of it but the changelog post will be ready before the blog post.